1.10/deploying-services/marathon-parameters.md

```diff
@@ -44,7 +44,7 @@ The container information.
 - **portMappings** The port mappings between host and container.  A port mapping is a tuple that contains a host port (`hostPort`), container port (`containerPort`), service port (`servicePort`), and protocol (`protocol`).  Port mappings are similar to passing `-p` into the Docker command line to specify a relationship between a port on the host machine and a port inside the container. 
 
     - **containerPort** The container port (e.g., `8080`).
-    - **hostPort** The host port (e.g., `0`). The default value is `0`. In `USER` mode, the hostPort is not required, but if left unspecified Marathon will not randomly allocate a port.
```

Comment:
do you mean the `networks.container` parameter?

Reply:
Not sure what you are referring to in that comment. Also, there is no `networks.container` parameter, only `networks.mode`, which can be set to `container` or `container/bridge`.

Reply:
OK, NM. I found "networks" confusing and thought it was a parameter maybe. Maybe remove "networks" to read, "In container mode...."

Reply:
@sascala `networks` **is** a parameter. It has three subparameters: `mode`, which can be host, container, or container/bridge.